shell/platform/android/test/io/flutter/embedding/engine/systemchannels/PlatformChannelTest.java

```diff
@@ -113,6 +115,61 @@ public void setSystemExclusionRectsSendsJSONExceptionOnIncorrectDataShape() thro
         );
     }
 
+    @Test
+    public void getSystemExclusionRectsSendsExclusionRectsToFrameworkOnSuccess() throws JSONException {
+        DartExecutor dartExecutor = mock(DartExecutor.class);
+        PlatformChannel platformChannel = new PlatformChannel(dartExecutor);
+        PlatformMessageHandler platformMessageHandler = mock(PlatformMessageHandler.class);
+        platformChannel.setPlatformMessageHandler(platformMessageHandler);
+
+        int top = 0;
+        int right = 500;
+        int bottom = 250;
+        int left = 0;
+
+        ArrayList<Rect> expectedExclusionRects = new ArrayList<Rect>();
```

Comment:
Are these really the "expected" exclusion rects? This looks to me like `fakeExclusionRects`...

Reply:
Yeah, in my mind, I saw it as "these are the exclusion rects that we expect returned in this test". I'll update it to `fakeExclusionRects`, since that is clearer.